lib/util/cidr.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Util
+
+  # provides helper methods for interacting with CIDR network addresses 
+  module CIDR
+    class << self
+
+      # returns the formatted CIDR formatted string for a network
+      def format_cidr cidr
```

Comment:
I've been encouraging people to follow the rule "if you method returns something, name it after the thing being returned," so this would be `formatted_cidr`.  Then verb names are reserved for method with side effects.  Simple rule that imo really improves readability.

Reply:
Sure, no problem!